### DISCUSSION TITLE
Solved: Need your help, vote for the lesson 6 question on stackexchange

### DISCUSSION THREAD

**Discussion by n4n0b1t3** [smartcontractkit/full-blockchain-solidity-course-py#1070] [Q&A] [answered]
To finally solve one of our reoccurring issues in this discussion group I am trying to get help from the large community of StackExchange.com and StackOverflow.com. 
> https://ethereum.stackexchange.com/questions/121687/
> https://stackoverflow.com/questions/71126128/

The thing is, you can get much more attention to your question when users vote them up.
So please click the up triangle ;-)

_If you haven't created an account there YOU DEFINITELY SHOULD do it now, because those forums are best (albeit not always the friendliest)._

![image](https://user-images.githubusercontent.com/32838961/154135919-0421945c-3596-4cdb-b815-245cef35d8fd.png)

**Comment by cromewar** [comment, marked as answer]
It's pinned now ;)

**Reply by n4n0b1t3** [reply]
@cromewar OK, thank you. After creating a bug ticket on the brownie repository, I believe you can unpin this post. Thank you very much!
The issue pinning is a very cool feature.
